### DISCUSSION TITLE
On which file is the sha256 Hash

### DISCUSSION THREAD

**Discussion by Rohith0009** [ThioJoe/YT-Spammer-Purge#927] [Q&A] [answered]
Which File's Hash is the sha256 Hash File? Is it the exe or the py file?

**Comment by Firecul** [comment, marked as answer]
The sha256 that is on the releases refers to the .exe file

![explorer_Calculate_and_Verify_Checksum](https://user-images.githubusercontent.com/34950371/192377378-dd8bdd4b-3f4d-469d-a0e3-e82812940b6d.png)

**Reply by Firecul** [reply]
The py file can be verified by looking through it but the exe can't anywhere near as easily, so having the hash helps prove it is what it appears to be.
